### PR TITLE
Feature/merge

### PR DESCRIPTION
**Pull request by mikahpinegar**
- deleted unneeded json file
- made drainage class unselectable
- fix default crop sorting
- simplify logic for reducing opacity in crops that aren't recommended, remove unneeded activeCropDataShadow variable, and delete CropDataRender.js
- rename selectedCrops to selectedCropIds
- rename activeCrops to activeCropIds
- allow the user to proceed with no goals selected
- add all goals to the calendar view instead of average goal rating
- fix background of hessian fly free date
- remove unneeded props from CropCalendarView.js and CropTable.js
- rename CropTableListItem.js to RenderTableItems.js
- rename pullCropData to updateCropData to match redux naming convention
- simplify cropDataFormatter logic

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://4.156.182.169:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://4.156.182.169:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by YazidZidane — COMMENTED** [review]
The auto built app is not available because auth0 issues, I can run locally though. Also I think I can still manually select drainage class on Site Conditions page, but when I click next and return back, the drainage class would auto change.

File: src/pages/CropSelector/CropCalendarView/CropCalendarView.js [hunk 1]
```diff
@@ -39,61 +39,80 @@ const growthIcon = {
...
+const CropCalendarView = () => {
   // redux vars
-  const cropDataStateRedux = useSelector((stateRedux) => stateRedux.cropData);
+  const cropDataRedux = useSelector((stateRedux) => stateRedux.cropData.cropData);
   const selectedGoalsRedux = useSelector((stateRedux) => stateRedux.goalsData.selectedGoals);
   const councilShorthandRedux = useSelector((stateRedux) => stateRedux.mapData.councilShorthand);
   const ajaxInProgressRedux = useSelector((stateRedux) => stateRedux.sharedData.ajaxInProgress);
-  const selectedCropsRedux = useSelector((stateRedux) => stateRedux.cropData.selectedCrops);
+  const selectedCropIdsRedux = useSelector((stateRedux) => stateRedux.cropData.selectedCropIds);
+  const activeGrowthPeriodRedux = useSelector((stateRedux) => stateRedux.cropData.activeGrowthPeriod);
```
[the remark this answers is not in the document]
Comment (mikahpinegar) [reply src/pages/CropSelector/CropCalendarView/CropCalendarView.js:49]: yeah this is still needed but the data moved in the api response and broke during migration. there's an existing issue to fix it

**Comment by mikahpinegar** [comment]
> The auto built app is not available because auth0 issues, I can run locally though. Also I think I can still manually select drainage class on Site Conditions page, but when I click next and return back, the drainage class would auto change.

when i said unselectable i meant that you can deselect the api response and have no drainage class selected if desired not that it's no longer possible to change from the api response

**Pull request merged**